app/install/kubernetes.md

```diff
@@ -14,17 +14,24 @@ links:
   gh_repo: "https://github.com/Kong/kong-dist-kubernetes/"
 ---
 
-# Kubernetes Ingress Controller
+# Kubernetes Ingress Controller for Kong
 
-You can install Kong CE or EE using our official <a href="https://github.com/Kong/kubernetes-ingress-controller">Kubernetes Ingress Controller</a>.
+Install Kong Community Edition (CE) or Kong Enterprise Edition (EE) using the official 
+<a href="https://github.com/Kong/kubernetes-ingress-controller"> Kubernetes Ingress Controller</a>.
 
-You can learn more by checking out the <a href="https://github.com/Kong/kubernetes-ingress-controller/blob/master/README.md">README.md</a> file in the GitHub repository. To get up and running quickly, you can also follow the <a href="https://github.com/Kong/kubernetes-ingress-controller/tree/master/deploy">Minikube and Minishift tutorials</a> on the same repository.
+Learn more via the <a href="https://github.com/Kong/kubernetes-ingress-controller/blob/master/README.md">README
+</a>. To get up and running quickly, follow the 
+<a href="https://github.com/Kong/kubernetes-ingress-controller/tree/master/deploy">Minikube and Minishift tutorials</a>.
 
-If you are curious, you can also read the <a href="https://konghq.com/blog/kubernetes-ingress-controller-for-kong/">official announcment</a> on our Blog. Finally, if you experience any problem or have some feedback, please <a href="https://github.com/Kong/kubernetes-ingress-controller/issues">open a new issue</a> on GitHub.
+The [Kubernetes Ingress Controller for Kong launch announcement](https://konghq.com/blog/kubernetes-ingress-controller-for-kong/) 
+is on the [Kong Blog](https://konghq.com/blog/). 
```

Comment:
Having those two links in the same sentence looks confusing to me - if I click the second, I will reach the Blog's homepage. Instead of that, we could link directly to the announcement in the `[Kong Blog]` link.

Reply:
Thanks for the feedback - I'm going to leave it as is.